documents/functionnal_specifications/functional.md

```diff
@@ -0,0 +1,445 @@
+    - Critical **alerts** for disasters or urgent matters.
+
+    **4. Map Interaction Menu**:
+    - Allows access to national data, laws, industries, and global overlays.
+
+**HUD of the world map:**
+
+![WorldHUD](pictures/WorldMapHUD.png)
+
+**HUD of a country map:**
+
+![CountryHUD](pictures/CountryMapHUD.png)
+
+#### 2. Mechanics
+---
+
+- The HUD dynamically updates based on player actions and global events:
+  - **Funds and GDP**:
+    - **GDP** grows or shrinks depending on the nation's policies and industrial activities.
+    - Allocate a percentage of GDP to **scientific research** for technological advancements or sustainable solutions.
+  - **Scientific Research System**:
+    - Players can fund scientists to research **new technologies** or **climate solutions**.
+    - Completed research unlocks tools like **renewable energy technologies**, **carbon capture systems**, or **drought-resistant crops**.
+    - Research projects require time and are influenced by **funding levels** and **geopolitical stability**.
+  - **Planet Health Metrics**:
+    - Real-time feedback on environmental trends to help players assess the effectiveness of their strategies.
+    - Includes visible consequences like rising sea levels, desertification, or ecosystem restoration.
+  - **Alert Notifications**:
+    - Warns players of critical issues such as **natural disasters**, **political unrest**, or **scientific breakthroughs**.
+
+**Integration of GDP and Research**
+
+- GDP acts as the **financial backbone**:
+  - Players must maintain GDP to fund **research** while minimizing harm to the environment.
+  - Excessive industrial expansion may increase GDP temporarily but lead to higher **emissions** and **ecological degradation**.
+- The **scientific research system** challenges players to:
+  - Prioritize research areas based on current global challenges.
+  - Balance short-term GDP sacrifices with long-term sustainability goals. 
+
+
+
+### Pin & Icon System
+
+#### 1. Design
+---
+
+We have 4 categories :
+
+We have defined five main categories in agriculture, each representing a key area that affects the ecological status of a zone. These categories can be upgraded to reduce environmental impact and improve sustainability.
+
+| Agriculture | Description | Icon |
+|-------------|-------------|------|
+| **Livestock** | Represents the impact of livestock farming on the environment, including greenhouse gas emissions, water usage, and land degradation. Upgrades in this category may focus on sustainable livestock practices, waste reduction, and improved animal welfare. | ![godot](pictures/agriculture/icons8-panneau-bovins-60.png) |
+**Crops**            | Covers the production and management of various crops. The focus is on soil health, pesticide use, water conservation, and crop rotation. Upgrades here can lead to reduced chemical runoff and improved resilience to climate change. | ![godot](pictures/agriculture/icons8-champ-50.png) |
```

Comment:
337 : remove `The` before `focus is on`; `,` before `conservation`; replace `Upgrades here` with `Those improvements`; `reduced` with `reduce`; `improved` with `improve`